client/memez/projects/memez/src/app/views/layout/layout.component.ts

```diff
@@ -22,13 +23,14 @@ export class LayoutComponent implements OnInit {
     public log: LoginStore,
     public likeStore: LikeStore,
     public fs: FeedStore,
-    public ls: LayoutStore
+    public ls: LayoutStore,
+    public sus: SignupStore
   ) {
 
     (async () => {
         await this.us.getUsers();
         await this.ps.getPosts();
-        await this.likeStore.getLikes();
+        /*await this.likeStore.getLikes();*/
```

Comment:
remember what I told you about parallel vs sequential ?

All of these can go inside a Promise.all 

`await Promise.all([this.us.getUsers(), this.ps.getPosts(), this.likeStore.getLikes()])`

You might wanna wrap these in a try{}catch(e){} also to handle any errors and be notified if something goes wrong

Reply:
nice